lib/apparmortest.pm

```diff
@@ -35,7 +36,8 @@ our $audit_log = "/var/log/audit/audit.log";
 sub aa_disable_stdout_buf {
     my ($self, $app) = @_;
     if (is_sle('<15') or is_leap('<15.0')) {    # apparmor < 2.8.95
-        assert_script_run "sed -i '/use strict;/a \$|=1;' $app";
+        assert_script_run "sed -i '/use strict
```

Comment:
Check your script

Reply:
:see_no_evil: thanks! Will push an updated version in a minute